password_security/views/password_security.xml

```diff
@@ -4,8 +4,8 @@
     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 -->
 <odoo>
-    <template id="assets_common" inherit_id="web.assets_common" name="Password Password Common Assets">
-        <xpath expr="." position="inside">
+    <template id="assets_common" inherit_id="web.assets_common" name="Password Security Common Assets">
+        <xpath expr="//script[@src='/auth_password_policy/static/src/js/password_gauge.js']" position="after">
```

Comment:
You probably don't need this change. Undo it please. As it was before, it will just apend your file to the end, and the order will be resolved by addon dependency order, so let's just KISS.